sparkbox/commit-colors
# Golang rewrite

Rewrites commit-colors in Golang, which means:

- It's dependency free. ☯️ 
- It's cross-platform. 🖥️ 
- Easier installs via homebrew. 🍻 

Other changes:
- 1600+ more color names. 🌈 
- It removes the optional template feature for now. 🗑️ 
- Better input checking and error handling. 🔥 
- Tests. ✅ 

The homebrew publish will happen once this PR is merged and I've pushed up a tag for version `2.0.0`.

## To test this:

1. Pull down the branch `git checkout -t origin/golang-rewrite`
2. Make sure you have [installed Golang](https://golang.org/doc/install).
3. Run the project (`make run`) and confirm that it gives you a white color swatch.
4. Run tests (`make test`) and confirm that there are no failing tests.
